presto-spi/pom.xml

```diff
@@ -48,6 +48,12 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
We cannot have guava dependency in SPI. Just use java utils

Reply:
Since we can't have `guava` in `presto-spi` and `presto-common`, and we need to use md5 functionality.
Currently we can either create a new project `presto-hash` which wasn't favored last time, or moving it to `presto-main`